tox.ini

```diff
@@ -4,18 +4,18 @@ minversion = 1.8
 envlist =
     py38-lint
     py38-docs
-    py{37,38,39,310}-dj32
-    py{38,39,310}-dj{40,41}
-    py{37,38,39,310}-fl{012,10,11,20,21,22}
-    py{37,38,39}-s20
-    py{37,38,39,310}-s{21,22}
+    py{37,38,39,310,311}-dj32
+    py{38,39,310,311}-dj{40,41}
```

Comment:
nit: could add [Django 4.2](https://docs.djangoproject.com/en/4.2/releases/4.2/), the LTS version released in April 2023. However, that could be a PR after this one, and would require at least a few more lines in this file.

Reply:
This was pretty easy to pull in, so I went ahead and did it [3dccd82](https://github.com/mozilla-services/python-dockerflow/pull/76/commits/3dccd82419b75150a7e62dfb172fa602fac6b7c7)